chromaui/addon-visual-tests
# UI to indicate new story and tests (modes)

This adds the UI for indicating if the current story is a new story, or if the current snapshot is a new test mode. 
It also disables the toggle snapshot button if there is no baseCapture, and [makes the footer and header sticky](https://linear.app/chromaui/issue/AP-3679/addon-header-and-footer-should-be-sticky)

To test:
 1. View storybook for changes.
 2. View the build for this branch by the addon on a new story [like this one](http://localhost:6006/?path=/story/screens-visualtests-snapshotcomparison--story-added)
     3. New story will have no toggle baseline either.
 4. Create a new story. It should show a UI to trigger a new story.


<details>
  <summary>📦 Published PR as canary version: <code>0.0.96--canary.101.301375d.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.96--canary.101.301375d.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.96--canary.101.301375d.0
  ```
</details>
